docs/site/Application-generator.md

```diff
@@ -103,3 +103,65 @@ greeted with a message.
 Once the application has been created, additional generators such as
 [controller generator](Controller-generator.md) can be run from the
 application's root directory to further scaffold the application.
+
+
+### Dockerfile Considerations
+
+The Dockerfile that is generated by the `--docker` option is based on a `slim` version of a `node` Docker image.
+
+Here are the first two lines of the [Dockerfile](https://github.com/strongloop/loopback-next/blob/master/packages/cli/generators/app/templates/Dockerfile): 
+
+```
+# Check out https://hub.docker.com/_/node to select a new base image
+FROM node:10-slim
+```
+
+As the comment suggests, the Docker image name is only a default, and you are encouraged to select an appropriate base image for your application.
+
+If your application has a dependency on a node module that requires and invokes build tools during `docker build` (for example : [loopback-connector-db2](https://github.com/strongloop/loopback-connector-db2) ), an error may occur due to missing libraries.
+
+```
+...
```

Comment:
This error message is very long and hides the actual advice to use `node:10` image. Can you please find a way how to move the advice above the error message? For example:

```markdown
If your application has a dependency on a node module that requires and invokes build tools 
during `docker build` (for example : [loopback-connector-db2](https://github.com/strongloop/loopback-connector-db2) ), 
an error may occur due to missing libraries.

Using the Docker image `node:10`, however, will allow the `docker build` step to complete successfully.

Example compilation error:

[here goes the long text]
```